### PR TITLE
RRD - Refactor Invariants Module to Match Blue Style Guide

### PR DESCRIPTION
**Pull request by RandyRDavila**
# Pull Request Checklist

This PR refactors the code within the `Invariants` module to match the[ blue style of writing Julia code ](https://github.com/invenia/BlueStyle). 

Thank you for contributing to `GraphProperties.jl`!

## Documentation and Style

- [x] I have added docstrings to any new functions or types, as applicable.
- [x] My code follows the project's coding style and guidelines.

## New Features and Code Changes

- [x] If you have added a new algorithm or function, you have also included corresponding unit tests.
- [x] I confirm that any code added is my own work and is not copied from other sources.

## Review Process

- [x] I have requested a review from at least one other contributor.

## Additional Notes

- Please describe any additional changes or information relevant to this PR.

---

### Developer's Declaration:

By submitting this Pull Request, I confirm the following:

- [x] The code I am submitting is my own work and I have the rights to use and distribute it.
- [x] I confirm that the code I have added is not copied from other sources and that I have appropriately cited or acknowledged any external code or references used.
- [x] I understand that my code will be reviewed and any changes requested must be addressed.

## PR Type

What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no API changes)
- [x] Documentation content changes
- [ ] Unit Testing
- [ ] Other... Please describe:

## What is the new behavior (if this is a feature change)?

- [ ] Yes
- [x] No

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Invariants/degree_sequence_invariants/annihilation.jl [hunk 1]
```diff
@@ -18,9 +18,8 @@ function compute(
     n = Graphs.nv(g)
 
     for i in reverse(1:n)
-        if sum(D[1:i]) <= m
-            return i
-        end
+        isum(D[1:i]) <= m && return i
```
Comment (joshyaffee) [inline src/Invariants/degree_sequence_invariants/annihilation.jl:21]: I searched `isum` in the REPL's help mode and didn't find anything. Do you mean `sum`?

File: src/Invariants/drawing/draw_optimal_sets.jl [hunk 2]
```diff
@@ -11,12 +11,14 @@ function draw_optimal_set(
...
     # Plot the graph with the color scheme
-    gplot(
+    plot = gplot(
         g,
         nodefillc=nodecolors,
         nodelabel=Graphs.vertices(g),
         layout=stressmajorize_layout
     )
+
+    return plot
```
Comment (joshyaffee) [inline src/Invariants/drawing/draw_optimal_sets.jl:21]: Maybe specify in the signature that this method returns type ::Compose.Context

File: src/Invariants/drawing/draw_optimal_sets.jl [hunk 3]
```diff
@@ -35,10 +37,12 @@ function draw_optimal_set(
...
+    plot = gplot(
         g,
         edgestrokec=edgecolors,
         nodelabel=Graphs.vertices(g),
         layout=stressmajorize_layout
     )
-end
\ No newline at end of file
+
+    return plot
```
Comment (joshyaffee) [inline src/Invariants/drawing/draw_optimal_sets.jl:47]: same as above

**Review by joshyaffee — APPROVED** [review]
I left feedback in individual comments. I think you should be good to merge after reviewing them.

File: src/Invariants/degree_sequence_invariants/annihilation.jl (hunk 1 above)
Comment (RandyRDavila) [reply src/Invariants/degree_sequence_invariants/annihilation.jl:21]: Good catch!

**Pull request merged**